.eslintignore

```diff
@@ -1 +1 @@
-src/__tests__/*
\ No newline at end of file
+src/**/__tests__/*
```

Comment:
Allow keeping tests closer to the things they test.

Reply:
We can also have the test in their associated component folder and name the file .test.js or .spec.js. Should discuss with @timcosgrove  and @amponce

Reply:
Sounds good to me 😃  Along those lines, we might differentiate with `.unit.test.js`, `.e2e.test.js`, etc for running different kinds of tests in different passes, using different settings, etc.